.github/ci/packages-jammy.apt

```diff
@@ -0,0 +1,47 @@
+freeglut3-dev
+libbenchmark-dev
+libfreeimage-dev
+libglew-dev
+libgz-cmake3-dev
+libgz-common5-dev
+libgz-fuel-tools9-dev
+libgz-gui8-dev
+libgz-math7-eigen3-dev
+libgz-msgs10-dev
+libgz-physics7-dev
+libgz-plugin2-dev
+libgz-rendering8-dev
+libgz-sensors8-dev
+libgz-tools2-dev
+libgz-transport13-dev
+libgz-utils2-cli-dev
+libogre-1.9-dev
+libogre-next-2.3-dev
+libprotobuf-dev
+libprotoc-dev
+libsdformat14-dev
+libtinyxml2-dev
+libxi-dev
+libxmu-dev
+libpython3-dev
+python3-distutils
```

Comment:
I believe packages from both this file and `packages.apt` will be installed (see [action-gz-ci/entrypoint.sh](https://github.com/gazebo-tooling/action-gz-ci/blob/focal/entrypoint.sh#L102)), so I think this file just needs this one line with `python3-distutils`